orchestrator/api/admin_plugins.py

```
@@ -0,0 +1,474 @@
+            plugin_id=str(plugin.id),
+            scan_id=str(plugin.security_scan_id) if plugin.security_scan_id else None,
+            slug=plugin.slug,
+            name=plugin.name,
+            version=plugin.version,
+            security_status=plugin.security_status or "pending",
+            approval_status=plugin.approval_status or "pending",
+            status="scanning",
+        )
+
+    except ValueError as e:
+        raise HTTPException(status_code=400, detail=str(e))
```

Ruff (0.14.14)

[warning] Within an `except` clause, raise exceptions with `raise ... from err` or `raise ... from None` to distinguish them from errors in exception handling

(B904)